src/posix/platform/udp.cpp

```diff
@@ -320,7 +334,7 @@ otError otPlatUdpConnect(otUdpSocket *aUdpSocket)
         // We create new socket to disconnect.
         SuccessOrExit(error = otPlatUdpClose(aUdpSocket));
         SuccessOrExit(error = otPlatUdpSocket(aUdpSocket));
-        SuccessOrExit(error = otPlatUdpBind(aUdpSocket));
+        SuccessOrExit(error = otPlatUdpBind(aUdpSocket, false));
```

Comment:
```suggestion
        SuccessOrExit(error = otPlatUdpBind(aUdpSocket, ???));
```
How do we know this socket was previous binded to Thread only or not?